_episodes/github-pages.md

```diff
@@ -168,8 +169,8 @@ We are now ready to start adding more content to our website. Let's do some exer
 > >
 > >        Twitter: [@my_research_project](https://twitter.com/my_research_project)
 > >
-> >     Note how we used various Markdown syntax: quoted text ('>'), italic font ('*') and external links
-> >     (square '[]' and round brackets '()' for mailto and regular Web URLs).
+> >     Note how we used various Markdown syntax: quoted text (`>`), italic font (`*`) and external links
+> >     (square `[]` and round brackets `()` for mailto and regular Web URLs).
```

Comment:
```suggestion
> >     (a combination of square `[]` and round brackets `()` containing the link text and mailto or regular Web URLs respectively).
```

I think this is clearer - the original wording could potentially be confusing as a learner might read it as meaning "use square brackets for mailto and regular brackets for Web URLs."